packages/client/src/checkout/types/postDraftOrders.types.ts

```diff
@@ -0,0 +1,15 @@
+import type { Config } from '../../types/index.js';
+import type { DraftOrder } from './draftOrder.types.js';
+
+export type PostDraftOrdersData = {
+  orderId: string;
+  customerId?: string;
+  paymentMethods?: [string];
+};
+
+export type PostDraftOrders = (
+  data: PostDraftOrdersData,
+  config?: Config,
+) => Promise<PostDraftOrdersResponse>;
+
+export type PostDraftOrdersResponse = DraftOrder;
```

Comment:
I would move this line to be before `PostDraftOrders` for readability.

Reply:
done 👍